specification/logic/resource-manager/readme.md

```diff
@@ -39,6 +39,15 @@ directive:
 
 ```
 
+### Tag: package-2019-05
+
+These settings apply only when `--package-2019-05` is specified on the command line.
```

Comment:
--tag=package-2019-05

Reply:
Fixed but still see the readme errors. What is the issue now?